src/VisualStudio/Core/Test.Next/Services/SolutionServiceTests.cs

```diff
@@ -47,14 +47,14 @@ public async Task TestCreation()
             var assetProvider = await GetAssetProviderAsync(workspace, remoteWorkspace, solution);
 
             var solutionChecksum = await solution.State.GetChecksumAsync(CancellationToken.None);
-            var synched = await remoteWorkspace.GetTestAccessor().GetSolutionAsync(assetProvider, solutionChecksum, fromPrimaryBranch: false, workspaceVersion: -1, CancellationToken.None);
+            var synched = await remoteWorkspace.GetTestAccessor().GetSolutionAsync(assetProvider, solutionChecksum, updatePrimaryBranch: false, workspaceVersion: -1, CancellationToken.None);
```

Comment:
renamed this to be clear what this flag is intending to convey. the flag it an actual request to the RemoteWorkspace to not only get the solution but update it's primary solution to point at that.